Author: Daniel Baumann <daniel@debian.org>
Description:
 doc/manpager seems to need gawk to produce proper manpages (Closes: #436523).

diff -Naurp barcode.orig/doc/manpager barcode/doc/manpager
--- barcode.orig/doc/manpager	2009-07-16 21:49:31.000000000 +0000
+++ barcode/doc/manpager	2009-07-16 21:57:26.000000000 +0000
@@ -1,4 +1,4 @@
-#! /usr/bin/awk -f
+#! /usr/bin/gawk -f
 # Copyright (c) 1998-2001 Alessandro Rubini
 
 BEGIN {IN=0}
